### DISCUSSION TITLE
Problem with mouse config (undefined node label 'mmv')

### DISCUSSION THREAD

**Discussion by joao-lobao** [urob/zmk-config#52] [Q&A] [answered]
Last month I could build my config with success but now even using the same commits that I used to build [here](https://github.com/joao-lobao/zmk-config/actions) with success I get this build error:

devicetree error: /keymap/layer_2: undefined node label 'mmv'

I have done no changes on the keymap includes nor west.yml

I see there is a recent commit that messes with mouse functionality. But I can't understand what is wrong. Is there missing a new config that I need to add but it is not on the README file or it can be other issue, is it just me?

Thanks in advance.

**Comment by urob** [comment, marked as answer]
Yes, there was a recent change in the mouse PR that I had pushed to my ZMK fork (which this repo builds against). The change requires an additional `include` for mouse movements to work:
```c
#include <behaviors/mouse_keys.dtsi>
```

This is the commit fixing it in my `zmk-config`: https://github.com/urob/zmk-config/commit/242f8d45ec86a5d1b68584cbc8bc67a288027cee

**Reply by joao-lobao** [reply]
That's it! Thanks for replying at lightning speed!
